src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -87,6 +89,14 @@ public OpenLoadFlowParameters setLowImpedanceBranchMode(LowImpedanceBranchMode l
         return additionalObservers;
     }
 
+    public boolean isPowerFactorConstant() {
+        return powerFactorConstant;
+    }
+
+    public void setPowerFactorConstant(boolean powerFactorConstant) {
```

Comment:
We use a fluent API here: could you please follow the same way to implement setters?